filament/src/details/Engine.h

```diff
@@ -516,6 +526,7 @@ class FEngine : public Engine {
             bool doFrameCapture = false;
         } renderer;
         matdbg::DebugServer* server = nullptr;
+        fgdbg::DebugServer* fgdbg = nullptr;
```

Comment:
you could also use a `std::unique_ptr<fgdbg::DebugServer>` to not have to worry about the `delete`.

Reply:
Cool I'll do that, didn't know where we stand on smart pointers :)